test/services/return-requirements/check.service.test.js

```diff
@@ -20,6 +21,11 @@ describe('Return Requirements - Check service', () => {
   let yarStub
 
   beforeEach(async () => {
+    Sinon.stub(ReturnRequirementsService, 'go').resolves({
```

Comment:
This feels necessary as i would have to create a licence and permitLicence for this test. 

I think it would be doing too much so have mocked it. 

The ReturnRequirementsService is tested using the database helpers so the output is covered.

Reply:
💯 agree!